docs.json

```diff
@@ -814,9 +814,7 @@
                         },
                         {
                             "title": "Bluetooth APIs",
-                            "sources": [{
-                                    "path": "docs/api/connectivity/bluetooth/BatteryService.md"
-                                },
+                            "sources": [
```

Comment:
Maybe we can adopt a more _logical_ organization: 
- BLE
- Gap
- GattServer
- SecurityManager
- Services
- GattClient (which is the least used)

What do you think ?